### ISSUE TITLE
Reply to messages on Telegram, WhatsApp, etc and react to calls

### ISSUE THREAD

**Issue by Algram** [holylobster/nuntius-linux#18]
Hey,
it would be really helpful if you could reply to various messages via your desktop and also take calls or decline them.

I don't know if this is doable, so I would like some input on that :)

**Comment by pbor** [comment]
Yes, this is definitely in the TODO list... we need to figure out how :)
